dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -169,9 +170,29 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
             // If the app was not launched due to ToastActivation, we will invoke the foreground handler.
             // Otherwise we store the EventArgs and signal to the Main thread
             auto pos{ commandLine.find(c_notificationActivatedArgument) };
-            if (pos == std::wstring::npos)
+            if (pos == std::wstring::npos) // !AppNotification
```

Comment:
Fix the comment please along the lines:
// If the Process was launched due to other Activation Kinds, we will need to re-route the payload to a new process if there are no registered event handlers.